doc/modules/svm.rst

```diff
@@ -222,6 +222,16 @@ calibrated using Platt scaling: logistic regression on the SVM's scores,
 fit by an additional cross-validation on the training data.
 In the multiclass case, this is extended as per Wu et al. (2004).
 
+Please note that when ``decision_function_shape='ovr'`` and ``n_classes > 2``,
```

Comment:
I think this should be the last paragraph: the paragraph above and the one below should be following each-other (they're both about calibrating probas and Platt scaling).